CONTRIBUTING.md

```diff
@@ -1,3 +1,13 @@
+## Contributing Guidance - v3
+
+* Target your pull requests to the **version-3** branch
+* Add/Update any docs articles related to your changes
+* Include a test for any new functionality and ensure all existing tests are passing by running `npm test`
+  * If you are fixing a bug, include a test that would have caught the bug you are fixing
+* Ensure tslint checks pass by typing `npm run lint`
+* Keep your PRs as simple as possible and describe the changes to help the reviewer understand your work
+* If you have an idea for a larger change to the library please [open an issue](https://github.com/pnp/pnpjs/issues) and let's discuss before you invest many hours - these are very welcome but want to ensure it is something we can merge before you spend the time :)
+
 ## Contributing Guidance - v2
```

Comment:
Let's remove references to V2, as we are no longer supporting v2 or publishing any new releases.

Reply:
Shouldn't I also remove v1 references?

Reply:
Yes please. :)